baybe/searchspace/discrete.py

```diff
@@ -60,6 +64,37 @@ class SubspaceDiscrete(SerialMixin):
     and thereby speed up construction. If not provided, the default hook will derive it
     from ``exp_rep``."""
 
+    def __str__(self) -> str:
+        if self.is_empty:
+            return ""
+
+        start_bold = "\033[1m"
+        end_bold = "\033[0m"
+
+        # Convert the lists to dataFrames to be able to use pretty_printing
+        par_df = parameter_cartesian_prod_to_df(self.parameters)
+        const_df = parameter_cartesian_prod_to_df(self.constraints)
+
+        # Put all attributes of the discrete class in one string.
+        discrete_str = (
```

Comment:
this reads pretty bad, did you consider to use a wide-line string via 
```
f"""
bla
newline dies das
searchspace info: {self.searchspaceinfo}
"""
```

Reply:
Thank you Martin! Fixed.